Client/Pages/Questions.razor

```diff
@@ -1,11 +1,14 @@
 ﻿@using Quanda.Shared.Models
 @using Quanda.Client.Components
+@using Quanda.Shared.Enums
 @page "/questions"
+@inject IQuestionsReposiotry QuestionsReposiotry
```

Comment:
According to docs, @Injected servicies are private and readonly, so they should be named e.g. _questionsRepository